db/migrate/20231025055844_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: :users }, index: true
+      t.references :post, foreign_key: true, index: true
+      t.text :text
+      t.timestamps
+    end
+
+    add_index :comments, :author_id
+    add_index :comments, :post_id
+  end
```

Comment:
- [ ] As required for the project and according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given to you, kindly ensure that all columns in the `posts` table are present. Right now, I noticed that you are missing the following columns ``comments_counter``, `likes_counter` and ``title`` and ensure that they have the corrct `data type`.